Assets/popUpObject.js

```diff
@@ -1,8 +1,8 @@
-/* eslint-disable no-constant-condition */
 /* eslint-disable no-unused-vars */
 const projectsInfo = [
   {
     name: 'Multi-Post Stories Gain+Glory',
+    namePop: 'Keeping track of hundreds of components',
```

Comment:
- [ ] [Optional] You should stick to the design as much as possible like color, font, and spacing. Please make your `x` buttons similar to the design and also stick to the design for buttons and use the same color for text. Please see the image below for reference
### Your design
![your](https://user-images.githubusercontent.com/81727988/209373456-c3627e91-ee4a-49b2-8fc0-2f805159d768.png)
### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/209373479-7d95debc-9f67-450e-a3b7-8ae053327d23.png)

Reply:
Hi Roshan, thank you for stating the issues 
My coding partner did in fact contribute to my project but I'm not sure why it is not showing in this pull request

Reply:
<img width="1250" alt="Screenshot 2022-12-23 at 8 23 51 PM" src="https://user-images.githubusercontent.com/90115863/209376308-5e3a6140-ab9c-4027-ad26-7ea7cdcadb52.png">

Hello Roshan,
Kindly check the comments a few days ago as shown in screenshot,
I appreciate your help.